Create 2022-11-17-monitoring-node-reboots.md (#135)
netdata/blog: blog/2022-11-17-monitoring-node-reboots.md

---
slug: monitoring-node-reboots
title: "How to monitor node reboots?"
description: "How to monitor node reboots?"
image: https://user-images.githubusercontent.com/96257330/202475049-22838a0b-73b1-485b-8416-5fd49d6ccb53.png
tags: [how-to,infrastructure-monitoring,reboots,nodes,shutdown,crash]
keywords: [how-to,infrastructure-monitoring,reboots,nodes,shutdown,crash]
authors: satya
---

> One of the most critical tasks of monitoring an infrastructure is to check the health of its servers / nodes.

Reviewer: ```suggestion
Monitoring the health and status of nodes and servers is a critical part of effective infrastructure monitoring.
```

<!--truncate-->

## How to monitor node reboots?

> One of the most critical tasks of monitoring an infrastructure is to check the health of its servers / nodes. In most cases this results in setting up a "Hardware manager" from the hardware vendor delivering these servers or setting up an SNMP (or similar) agent to continuously monitor the availability of the server and report when there is a reboot / failure.

Reviewer: Do we need to repeat this line?

Author: Resolved based on your suggestion below.
